src/lib/CookieJar.ts

```diff
@@ -136,3 +136,12 @@ export default class CookieJar {
         return Nimiq.BufferUtils.fromBase64(encodedWallets).length;
     }
 }
+
+namespace CookieJar { // tslint:disable-line no-namespace
+    export enum StatusFlags {
```

Comment:
If we use the plural form, then the type is `CookieJar.StatusFlags`. I really think we should use more singular for these enums.

Reply:
In my opinion it is fine for bit flags, as you can have more than one ` CookieJar.StatusFlags.KEY_MISSING | CookieJar.StatusFlags.HAS_PIN`. Generally it makes sense to use singular though, especially if the type is used regularly (rarely for bit flags).